app/src/androidTest/java/com/github/se/signify/ui/screens/LoginScreenTest.kt

```diff
@@ -32,39 +30,22 @@ class LoginScreenTest : TestCase() {
 
   @Before
   fun setUp() {
-    Intents.init()
-
     navigationActions = mock(NavigationActions::class.java)
     userSession = mock(UserSession::class.java)
 
     `when`(navigationActions.currentRoute()).thenReturn(Screen.AUTH.route)
+    Intents.init()
+    composeTestRule.setContent { LoginScreen(navigationActions, {}) }
```

Comment:
Good initiative adding this here. For such a small PR, I think you can afford to spill over into other tests a bit.